lang_php/parsing/parser_php.mly

```diff
@@ -544,6 +544,7 @@ function_body:
 async_opt:
```

Comment:
I would probably have renamed this to attribute_opt now that it's not just async.

Reply:
ohhh yes good point!